pyseir/cli.py

```diff
@@ -53,40 +54,37 @@ def _run_infer_rt(states: List[str], states_only=False):
         infer_rt.run_rt_for_fips(fips)
 
 
-def _run_mle_fits(states: List[str], states_only=False):
-    for state in states:
-        model_fitter.run_state(state, states_only=states_only)
-
-
-def _run_ensembles(states, ensemble_kwargs=dict(), states_only=False):
-    for state in states:
-        ensemble_runner.run_state(state, ensemble_kwargs=ensemble_kwargs, states_only=states_only)
-
-
 def _map_outputs(
-    states, output_interval_days=1, states_only=False, output_dir=None, run_mode="default"
+    states: List[pipeline.Region],
```

Comment:
Calling this `states` feels a bit misleading now? `state_regions` maybe? Understandable if that doesn't feel correct.

Reply:
Done